### PR TITLE
Initialize-project with components.

### PR DESCRIPTION
**Pull request by Daphineatim**
### In this milestone, the following were implemented

- Initialized React app.
-  Added [React Router](https://reactrouter.com/web/guides/quick-start)
-  Set two Routes and Links for the app's navigation:

- [ ] Books - the default view
- [ ] Categories

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "babel-install": "^2.1.0",
+    "eslint-config-airbnb-base": "^15.0.0",
```
Comment (GzimAsani) [inline package.json:11]: - [x] The app is throwing this error, kindly fix it 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @Daphineatim                :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job adding props properly:+1:  :+1: 

 ## Required Changes 

- [x] On your app.js file kindly update the route for books because it is not working
- [x] On readme file kindly add the author
- [x] Check the comments below :arrow_double_down: 







 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (Daphineatim) [reply package.json:11]: Resolved

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Daphineatim  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes as requested by previous reviewer 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**